README.md

```diff
@@ -7,8 +7,8 @@ It reuses the classical types and statements from `openqasm3`.
 
 ### Working with submodule
```

Comment:
**issue**: It's no longer a submodule, so the README needs updating.

Reply:
Should we not also get rid of the OpenQASM submodule?

Reply:
Ah, I should have read [your comment](https://github.com/openqasm/openpulse-python/pull/2#issue-1368343016) first.

Reply:
Since we've pinned the OpenQASM package version, should we remove the submodule and just copy-paste the OpenQASM grammar files? That's all we need right?